### PR TITLE
Adding an acronym lookup because lord knows do i need it

### PR DESCRIPTION
**Pull request by Mutmatt**
#### Changes in this PR
This pull request adds the ability to lookup acronyms. 

#### Possible drawbacks
I used a pattern for including multiple keys that may or may not match with what we collectively want. I used a pipe delimited list (e.g. CC|5s|CLS)

#### Applicable Issues
TODO adding acronym lookup

#### Checklist

- [x] Code passes eslint check/ is valid
- [X] Code is commented and readable

### CODE REVIEW

File: commands/acronyms.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    async run(client, message, [ acronym ], options) { // eslint-disable-line no-unused-vars
+        const acronymsLookup = client.acronyms;
+        const acronyms = Object.keys(acronymsLookup);
+
+        if (!acronym.length) {
+            return message.channel.send(message.language.get("COMMAND_ACRONYMS_INVALID", acronyms.join(", ")));
+        }
+
+        const lookupList = acronym.split("|").map(a => a.toLowerCase());
```
Comment (jmiln) [inline commands/acronyms.js]: To get rid of extra spaces before or after that'd make it not match
```suggestion
        const lookupList = acronym.split("|").map(a => a.trim().toLowerCase());
```

File: languages/en_US.js [hunk 2]
```diff
@@ -206,6 +206,29 @@ module.exports = class extends Language {
                 ]
             },
 
+            //Acronym Command
+            COMMAND_ACRONYMS_INVALID: (prefix) => `Acronym lookup provided not available for blank`,
```
Comment (jmiln) [inline languages/en_US.js]: Don't need the `(prefix) =>` on these lines since it's not used.

File: languages/en_US.js [hunk 3]
```diff
@@ -206,6 +206,29 @@ module.exports = class extends Language {
...
+            COMMAND_ACRONYMS_NOT_FOUND: (prefix) => `Acronym could not be found`,
+            COMMAND_ACRONYMS_FOUND: (acronyms, meanings) => {
```
Comment (jmiln) [inline languages/en_US.js]: This should be in the main file, then just pass a string in here. That way it's not confusing for people trying to translate it, who aren't so familiar with the code side of  things

Comment (Mutmatt) [reply languages/en_US.js]: Should the acronyms responses be a translatable list?

Comment (Mutmatt) [reply languages/en_US.js]: Moved this and added a comment. We could do something like 

```javascript
const acronymMeaning = message.language.get("COMMAND_ACRONYM_" + matchingItems[i]);
acronymMeaningMessage += `**${matchingItems[i]}**: ${acronymMeaning}`;
```

or change the json to have 
```javascript
{
    "CC": "COMMAND_ACRONYM_CC",
...
}
```

**Pull request merged**